csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -94,6 +94,31 @@ public static SessionOptions MakeSessionOptionWithNupharProvider(String settings
             return options;
         }
 
+        /// <summary>
+        /// A helper method to construct a SessionOptions object for ROCM execution.
+        /// Use only if ROCM is installed and you have the onnxruntime package specific to this Execution Provider.
+        /// </summary>
+        /// <returns>A SessionsOptions() object configured for execution on deviceId=0</returns>
+        public static SessionOptions MakeSessionOptionWithRocmProvider()
+        {
+            return MakeSessionOptionWithRocmProvider(0);
+        }
+
+        /// <summary>
+        /// A helper method to construct a SessionOptions object for ROCM execution.
+        /// Use only if ROCM is installed and you have the onnxruntime package specific to this Execution Provider.
+        /// </summary>
+        /// <param name="deviceId"></param>
+        /// <returns>A SessionsOptions() object configured for execution on deviceId</returns>
+        public static SessionOptions MakeSessionOptionWithRocmProvider(int deviceId = 0)
+        {
+            CheckRocmExecutionProviderDLLs();
```

Comment:
The error below reported from CI is related to CheckRocmExecutionProviderDLLs. 

##[error]csharp\src\Microsoft.ML.OnnxRuntime\SessionOptions.cs(115,13): Error CS0103: The name 'CheckRocmExecutionProviderDLLs' does not exist in the current context